waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -784,9 +787,21 @@ proc mountRlnRelayDynamic*(node: WakuNode,
     rlnIndex = regIndexRes.value
     debug "peer is successfully registered into the membership contract"
   else:
+    info "Peer is already registered to the membership contract"
     keyPair = memKeyPair.get()
     rlnIndex = memIndex.get()
 
+  #Write KeyPair
+  let kp = %keyPair
+  writeFile("keyPair.txt", pretty(kp))
+
+  #Write rlnIndex
+  let ri = %rlnIndex
+  writeFile("rlnIndex.txt", pretty(ri))
+
+  #Since the files are stored as a raw text file, it is highly susceptible to theft.
+  #The files needs some encryption to resolve this.
```

Comment:
Thanks for the suggestion. I will have a look at this. There has also been a request to look at the keystore implemented in Nimbus by other contributors of nwaku. Since this pull request is focusing on "persistence" of the keys and not the "security" aspect of it, can you review it from the former perspective? I have opened another issue to evaluate different encryption methods. https://github.com/status-im/nwaku/issues/1041

Reply:
Of course: my comment was indeed just informative and not relative to this PR.